component/authenticator/src/main/java/org/wso2/carbon/identity/authenticator/mepin/MepinTransactions.java

```diff
@@ -240,6 +268,9 @@ protected String getTransaction(String url, String transactionId, String clientI
             if (inputStream != null) {
                 inputStream.close();
             }
+            if (outputStream != null) {

```

Comment:
Can't we use try-with-resources to create outputStream? If so we don't need to close the outputStream here

Reply:
Same reason as above.